docs/src/pages/components/avatar.mdx

```diff
@@ -36,12 +41,19 @@ The `Avatar` component renders a generic avatar graphic, or a stylized label.
     <tbody>
         <tr>
             <td>label</td>
-            <td>Replace the default avatar graphic with the first two characters of the provided
-                string.  Letters will be automatically capitalized.
+            <td>
+                Replace the default avatar graphic with the first two characters
+                of the provided string. Letters will be automatically
+                capitalized.
             </td>
             <td>string</td>
             <td>-</td>
         </tr>
+        <tr>
+            <td>src</td>
+            <td>Replace the default avatar graphic with an image.</td>
+            <td>string</td>
+            <td>-</td>
+        </tr>
     </tbody>
```

Comment:
Since you added `alt`, need to add it to the api list. It'd be good to put it in your example too.

Reply:
Did both.